src/Utilities/hooks/useTableTools/useExportHelpers.js

```diff
@@ -0,0 +1,6 @@
+export const linkAndDownload = (data, filename) => {
```

Comment:
I think we can utilise this function form the `fec/utils` package: https://github.com/RedHatInsights/frontend-components/blob/master/packages/utils/src/helpers/helpers.ts#L14

Reply:
Done